### PR TITLE
fix: set valid label value for nested gitlab repository

### LINKED ISSUE #819: Invalid label name for nested gitlab repos in jx source repository
```
The SourceRepository "jx-gitlab-test-cluster-gitlab-import-test-1" is invalid: metadata.labels: Invalid value: "cluster/gitlab-import-test-1": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```
Related to https://github.com/jenkins-x/jx/issues/7981

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

fixes #819

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@ approvers:
 - jstrachan
 - rajdavies
 - chrismellard
+- ankitm123
```
Comment (ankitm123) [inline OWNERS:6]: Can help with reviewing/approving PRs ...

**Comment by marziman** [comment]
@ankitm123 it stucked somehow. I believe the other ones got reviewed.

**Comment by ankitm123** [comment]
/hold
Putting on hold to upgrade packages in go.mod file ... no need to make separate PRs imo ...

**Comment by ankitm123** [comment]
No issues running tests locally :thinking:

File: go.mod [hunk 2]
```diff
@@ -14,10 +14,9 @@ require (
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/imdario/mergo v0.3.12
 	github.com/jenkins-x-plugins/jx-charter v0.0.28
-	github.com/jenkins-x/gen-crd-api-reference-docs v0.1.6 // indirect
```
Comment (ankitm123) [inline go.mod:17]: Not sure why this was added? from `make docs`?

File: go.mod [hunk 3]
```diff
@@ -14,10 +14,9 @@ require (
...
-	github.com/jenkins-x/go-scm v1.10.10
+	github.com/jenkins-x/go-scm v1.10.11
 	github.com/jenkins-x/jx-api/v4 v4.3.0
-	github.com/jenkins-x/jx-helpers/v3 v3.1.0
+	github.com/jenkins-x/jx-helpers/v3 v3.1.1
 	github.com/jenkins-x/jx-kube-client/v3 v3.0.2
 	github.com/jenkins-x/jx-logging/v3 v3.0.6
 	github.com/jenkins-x/lighthouse-client v0.0.233
```
Comment (ankitm123) [inline go.mod:22]: Still needs the lighthouse release pipeline to succeed before I can bump this ...

Comment (ankitm123) [reply go.mod:22]: I think that can be skipped for now, jx-helpers and go-scm were more important.

**Pull request merged**